app/globals.py

```diff
@@ -62,3 +65,23 @@ def get_answer_store(user):
 def get_completed_blocks(user):
     questionnaire_store = get_questionnaire_store(user.user_id, user.user_ik)
     return questionnaire_store.completed_blocks
+
+
+def get_dynamodb():
+    if not is_dynamodb_enabled():
+        return
+
+    db = getattr(g, '_dynamodb', None)
+    if db is None:
+        # Number of additional connection attempts
+        config = Config(retries={'max_attempts': 1})
+
+        dynamodb = boto3.resource(
+            'dynamodb', endpoint_url=current_app.config['EQ_DYNAMODB_ENDPOINT'], config=config)
+
+        db = g._dynamodb = dynamodb
```

Comment:
Would `return g._dynamodb = dynamodb` work?

Reply:
Yes, but that line only gets reached if the dynamo resource object wasn't already on the context object. So we'd have two returns:

```
db = getattr(g, '_dynamodb', None)
if db is None:
   dynamodb = stuff()
   return g._dynamodb = dynamodb
return db
```

Reply:
Ah yeh I suppose its tidier. Just seems odd to set a variable that we don't need.